### PR TITLE
Author game unit tests

### LINKED ISSUE #19: [1.0] Add unit tests for implemented methods #Team Member 3
- Add unit tests for all implemented methods.

### PR DESCRIPTION
**Pull request by torobucci**
- [x] #19

### CODE REVIEW

File: Rspec/spec/author_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  describe '#initialize' do
+    it 'should receive and set first and last names of author' do
+      expect(author.first_name).to eql(first_name)
+      expect(author.last_name).to eql(last_name)
+    end
+    it 'first and last names should be strings' do
+      expect(author.first_name).to be_a(String)
+      expect(author.last_name).to be_a(String)
+    end
+  end
```
Comment (Odongo006) [inline Rspec/spec/author_spec.rb:19]: When I run 'Rspec filename_rspec.rb' I should see the tests running with examples and tests passed, I think you need to fix that to ensure that your unit testing is complete

**Review by Odongo006 — DISMISSED** [review]
Hello @torobucci 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlight

- Good use of gitflow ✔️
- No Linters Error ✔️
- Descriptive pull request ✔️
## Required Changes ♻️
Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews.**
_You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

File: Rspec/spec/author_spec.rb (hunk 1 above)
Comment (ReemMohamedAbdelfatah) [reply Rspec/spec/author_spec.rb:19]: Hello @torobucci 
It gives this error "No examples found" despite I installed the required packages after switching to the branch.
noexamples

**Review by ReemMohamedAbdelfatah — APPROVED** [review]
# Hi @torobucci 👋
You have done very well in this PR. 👍 
Congratulations! 🎉 

FlashingAnimatedGIF

Nothing to say except it's time to merge it.
 
## Highlights 🌟
✅ Correct Github WorkFlow.
✅ No Linter errors.
✅ Good commit messages 
✅ Good Code Structure 
✅ Good documentation

## Optional changes 💯 

- None

*I strongly recommend you take every comment with the [OPTIONAL] prefix into account as they can make your code better.* 

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the Issues thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

Happy coding! 💻

**Pull request merged**